understanding/reflow.html

```diff
@@ -120,14 +113,18 @@ <h3>横書き及び縦書きの言語でのスクロールの回避</h3>
                
                <h3>リフローと達成基準 1.4.4 テキストのサイズ変更の関係</h3>
                
-               <p>リフローの達成基準の焦点は、利用者が 2 方向にスクロールしなくてもズームインできるようにすることである。<a href="resize-text.html" class="sc">達成基準 1.4.4 テキストのサイズ変更</a>も適用することで、リフローの要件を同時に満たしながら、全てのテキストのサイズを少なくとも 200 ％に拡大することができる。利用者がズームインしたとき (又は小さなスクリーンで利用しているとき) にテキストのサイズが縮小された場合でも、200% の拡大率を得ることは可能であるべきである。例えば、ウィンドウの幅が1280px のときにテキストが 20px に設定されている場合、200 ％ズームでは最低 20px (デフォルトサイズの 200 ％) になるが、400 ％ズームでは (デフォルトの 100 ％表示の まだ 200 ％であるので) 10px に設定できる。全てのブレークポイントで 200 ％の拡大を可能にする必要はないが、何かしらの方法で 200 ％の拡大を可能にすべきである。</p>
-
-               <div class="note">
-                  <div role="heading" class="note-title marker" aria-level="4">訳注</div>
-                  <div>
-                     <p>「リフローと達成基準 1.4.4 テキストのサイズ変更の関係」の説明には不明瞭な箇所があるが、この箇所の翻訳は原文に忠実な形で行った。なお、最新の原文の該当箇所は大幅に加筆されている。Understanding WCAG 2.1 <a href="https://w3c.github.io/wcag/understanding/reflow.html#the-relation-of-reflow-to-the-success-criterion-1.4.4-resize-text">The relation of Reflow to the Success Criterion 1.4.4 Resize Text</a> もあわせて参照のこと。</p>
-                  </div>
-               </div>
+               <p>リフローの達成基準の焦点は、利用者が 2 方向にスクロールしなくてもズームインできるようにすることである。<a href="resize-text.html" class="sc">達成基準 1.4.4 テキストのサイズ変更</a>も適用することで、リフローの要件を同時に満たしながら、全てのテキストのサイズを少なくとも 200 ％に拡大することができる。ほとんどの実装では、テキストは拡大率が上がるにつれて拡大し続けることが予想されるため、利用者はテキストを最大400 ％まで  (そしてそれ以上) に拡大することができる。人々がズームインするときにテキストのサイズが一貫して増加しない実装 (小さな画面の用途に適応するためにメディアクエリに基づいてテキストが変換される場合など) では、テキストのサイズ変更の基準を満たすために、依然として 200％ の拡大することが可能でなければならない。</p>
+               
+               <p>例えば、デフォルトのブラウザ設定である100 ％ズームで、ウィンドウの幅が 1280px のときにテキストが20pxに設定されている場合、200 ％ズームで同じ 20px は、200 ％のテキストサイズを意味する。400 ％ズームでは、コンテンツ制作者はテキストサイズを 10px に設定することを決定してもよい。テキストは、デフォルトのブラウザ設定である 100 ％ズームと比較して、200 ％に拡大される。全てのブレークポイントで200％のテキスト拡大を達成する必要はないが、何らかの方法で 200 ％のテキスト拡大を実現できるべきである。</p>
+               
+            </section>
+            <section id="browsers-on-mobile-operating-systems">
+               
+               <h3>モバイルオペレーティングシステムのブラウザ</h3>
+               
+               <p>モバイルオペレーティングシステムのほとんどのブラウザは、デスクトップブラウザと同じようにリフローとズームを組み合わせていない。これらのモバイルブラウザは通常、デバイスの向きを変更するときにリフローをサポートする。コンテンツは新しいビューポート幅に調整される。しかし、これらのモバイルブラウザは、1.4.4 を達成するためにコンテンツを拡大することしかできない。ピンチジェスチャを使用してコンテンツを拡大したり、特定の列をダブルタップしてビューポートの幅を埋めたりするなど、リフローを単一の次元に制限しない方法でテキストのサイズを変更する。つまり、ほとんどのモバイルブラウザでズームされたコンテンツには、コンテンツ制作者が何をするかにかかわらず、2 次元のスクロールを伴うことを意味する。</p>
```

Comment:
```suggestion
               <p>モバイルオペレーティングシステムのほとんどのブラウザは、デスクトップブラウザと同じようにリフローとズームを組み合わせていない。これらのモバイルブラウザは通常、デバイスの向きを変更するときにリフローをサポートし、コンテンツは新しいビューポート幅に調整される。しかし、これらのモバイルブラウザは、1.4.4 テキストのサイズ変更を達成するためにコンテンツを拡大することしかできない。ピンチジェスチャを使用してコンテンツを拡大したり、特定のカラムをダブルタップしてビューポートの幅を埋めたりするなど、リフローを単一の次元に制限しない方法でテキストのサイズを変更する。つまり、ほとんどのモバイルブラウザでズームされたコンテンツには、コンテンツ制作者が何をするかにかかわらず、2 次元のスクロールを伴うことを意味する。</p>
```

Reply:
これ1.4.4. Resize Textとありますけど、ここのピリオドは文末のそれではないのでは…ということでさらに手を加えてみました。いかがでしょうか？

```suggestion
               <p>モバイルオペレーティングシステムのほとんどのブラウザは、デスクトップブラウザと同じようにリフローとズームを組み合わせていない。これらのモバイルブラウザは通常、デバイスの向きを変更するときにリフローをサポートし、コンテンツは新しいビューポート幅に調整される。しかし、これらのモバイルブラウザは、ピンチジェスチャを使用してコンテンツを拡大したり、特定のカラムをダブルタップしてビューポートの幅を埋めたりするなど、リフローを単一の次元に制限しない方法でしか 1.4.4 テキストのサイズを変を達成するためにコンテンツを拡大することができない。つまり、ほとんどのモバイルブラウザでズームされたコンテンツには、コンテンツ制作者が何をするかにかかわらず、2 次元のスクロールを伴うことを意味する。</p>
```